.circleci/config.yml

```diff
@@ -9,7 +9,7 @@ executors:
         default: false
```

Comment:
As we started migrating all repos to Github actions, it is required to  stick with circleci? (I really love circleci suggestion is to keep consistency)

Reply:
I'm fine with disabling circleci as from my experience github actions were working equally fine if not better (faster).

The browser tests are disabled on circleci as they didn't work properly. Don't remember exact details but it was an unresolved issue on their side, I believe.

Reply:
My commit message:
```
It's randomly failing chrome browser testing and since github
is also testing everything, we'll use that for now.
```

Reply:
I don't have admin rights to disable building on circleci. Can you do that @pi0? The link should be https://app.circleci.com/settings/project/github/nuxt-community/i18n-module?return-to=https%3A%2F%2Fapp.circleci.com%2Fpipelines%2Fgithub%2Fnuxt-community%2Fi18n-module